lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/BinarySpec.hs

```diff
@@ -127,6 +131,14 @@ spec = do
                             \the expected 3 bytes"
                         e `shouldContain` "getMessage"
 
+        it "Regression #1025" $ do
+            let bytes = BL.fromStrict $ unsafeFromHex
+                    "005200000002d5d800000000ffffffff000000003a2954d0068a79b7fa\
+                    \44d9b75d81a78e0c527b3060e923b28d6c981e9024ca28000000000000\
+                    \0000000000000000000000000000000000000000000000000000"
```

Comment:
@KtorZ, could you say a little bit about how you generated this hexadecimal string?

Perhaps a comment here would be useful to the reader.

Reply:
:+1: That's a block header. Took it from the integration `block0.bin` and changed the slot number to make sure if overflow (`fffffffff`)

Reply:
https://github.com/input-output-hk/cardano-wallet/compare/20c14256a1568a24240af01d4044f5e49b6a63dc..28436b01bd8c2e11ee3af18ff10835fcabaab565